### ISSUE TITLE
Themes may have templates in root directory, not templates directory

### ISSUE THREAD

**Issue by mglaman** [mglaman/retrofit-drupal#27]

**Comment by darrenoh** [comment]
Not too concerned about this one. It's a minor effort to move files to a location that works for both D7 and D10. This is similar to modules needing to namespace their class files for PSR4 discovery.

**Comment by mglaman** [comment]
True. It may not be a big issue. Especially since the diff should be minimal. But it'd be great for it to work. Or provide a warning
